doc/compe.txt

```diff
@@ -183,6 +208,7 @@ LUA CONFIG                                                  *compe-usage-ex-lua*
       vsnip = true;
       nvim_lsp = true;
       nvim_lua = true;
+      snippets = true;
```

Comment:
Hmm.... the `snippets` is very common name...

So I use `vim_lsp` for `vim-lsp` plugin (it has `lsp` namespace but because the name is too common)

In my personal opinion, the source name should be `snippets_nvim`.

What do you think about it?

Reply:
yeah, I was also thinking about that. I didn't thought using `_` instead of `.`, will change it in a second